Build Fails on Oracle Linux 6

Hi, 
I tried to compile the application on Oracle Linux 6, but make process fails somehow.
It worked perfectly on Oracle Linux 7 and Fedora 26.
Could you please help me to find out the cause and compile it?

Here is the error that I get : 
```
[  4%] Building CXX object dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/CMakeFiles/buffer_static.dir/src/CommonFWriteBufferFlusher.cpp.o
In file included from /usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:11:
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/include/CommonFWriteBufferFlusher.h:59: error: ‘__int64’ does not name a type
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/include/CommonFWriteBufferFlusher.h:61: error: ‘FilePosType’ was not declared in this scope
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/include/CommonFWriteBufferFlusher.h:61: error: template argument 2 is invalid
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/include/CommonFWriteBufferFlusher.h:61: error: template argument 5 is invalid
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp: In member function ‘virtual void Common::FWriteBufferFlusher::startMark()’:
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:96: error: ‘FilePosType’ was not declared in this scope
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:96: error: expected ‘;’ before ‘currentPos’
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:100: error: request for member ‘insert’ in ‘((Common::FWriteBufferFlusher*)this)->Common::FWriteBufferFlusher::mMarkIds’, which is of non-class type ‘Common::FWriteBufferFlusher::MarkIdToFilePos’
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:100: error: ‘currentPos’ was not declared in this scope
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp: In member function ‘virtual bool Common::FWriteBufferFlusher::jumpToMark(Common::IBufferFlusher::MarkId, bool)’:
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: ‘Common::FWriteBufferFlusher::MarkIdToFilePos’ is not a class type
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: ‘Common::FWriteBufferFlusher::MarkIdToFilePos’ is not a class type
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: ‘Common::FWriteBufferFlusher::MarkIdToFilePos’ is not a class type
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: ‘Common::FWriteBufferFlusher::MarkIdToFilePos’ is not a class type
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: ‘Common::FWriteBufferFlusher::MarkIdToFilePos’ is not a class type
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: ‘const_iterator’ is not a member of ‘Common::FWriteBufferFlusher::MarkIdToFilePos’
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:126: error: expected ‘;’ before ‘markIdIt’
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:127: error: ‘markIdIt’ was not declared in this scope
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:127: error: request for member ‘end’ in ‘((Common::FWriteBufferFlusher*)this)->Common::FWriteBufferFlusher::mMarkIds’, which is of non-class type ‘Common::FWriteBufferFlusher::MarkIdToFilePos’
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:133: error: ‘FilePosType’ was not declared in this scope
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:133: error: expected ‘;’ before ‘pos’
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:141: error: ‘pos’ was not declared in this scope
/usr/local/share/collada2gltf/dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/src/CommonFWriteBufferFlusher.cpp:145: error: request for member ‘erase’ in ‘((Common::FWriteBufferFlusher*)this)->Common::FWriteBufferFlusher::mMarkIds’, which is of non-class type ‘Common::FWriteBufferFlusher::MarkIdToFilePos’
make[2]: *** [dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/CMakeFiles/buffer_static.dir/src/CommonFWriteBufferFlusher.cpp.o] Error 1
make[1]: *** [dependencies/OpenCOLLADA/OpenCOLLADA/common/libBuffer/CMakeFiles/buffer_static.dir/all] Error 2
make: *** [all] Error 2

```

Regards.

I got following exception since gcc version of OL6 is 4.4.7. I can't update it either.
`cc1plus: error: unrecognized command line option "-std=c++11"
make[2]: *** [GLTF/CMakeFiles/GLTF.dir/src/Base64.cpp.o] Error 1
make[1]: *** [GLTF/CMakeFiles/GLTF.dir/all] Error 2`

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -5,6 +5,7 @@ Change Log
 
 ##### Fixes :wrench:
 * Upgrade to Draco 1.2.5 [#179](https://github.com/KhronosGroup/COLLADA2GLTF/pull/179)
+* Fix issue with exporting normal maps [#182](https://github.com/KhronosGroup/COLLADA2GLTF/pull/182)
 
 ### v2.1.1 - 2018-04-04
 

--- a/GLTF/src/GLTFAsset.cpp
+++ b/GLTF/src/GLTFAsset.cpp
@@ -274,6 +274,12 @@ std::vector<GLTF::Texture*> GLTF::Asset::getAllTextures() {
 					uniqueTextures.insert(values->specularTexture);
 				}
 			}
+			if (values->bumpTexture != NULL) {
+				if (uniqueTextures.find(values->bumpTexture) == uniqueTextures.end()) {
+					textures.push_back(values->bumpTexture);
+					uniqueTextures.insert(values->bumpTexture);
+				}
+			}
 		}
 		else if (material->type == GLTF::Material::PBR_METALLIC_ROUGHNESS) {
 			GLTF::MaterialPBR* materialPBR = (GLTF::MaterialPBR*)material;
@@ -419,7 +425,8 @@ void GLTF::Asset::removeUnusedSemantics() {
 				if (semantic.find("TEXCOORD") != std::string::npos) {
 					std::map<std::string, GLTF::Accessor*>::iterator removeTexcoord = primitive->attributes.find(semantic);
 					if (semantic == "TEXCOORD_0") {
-						if (values->ambientTexture == NULL && values->diffuseTexture == NULL && values->emissionTexture == NULL && values->specularTexture == NULL) {
+						if (values->ambientTexture == NULL && values->diffuseTexture == NULL && values->emissionTexture == NULL && 
+								values->specularTexture == NULL && values->bumpTexture == NULL) {
 							std::map<std::string, GLTF::Accessor*>::iterator removeTexcoord = primitive->attributes.find(semantic);
 							primitive->attributes.erase(removeTexcoord);
 							removeAttributeFromDracoExtension(primitive, semantic);

